client/src/components/AreasOfFocus/styles.css

```diff
@@ -69,6 +69,19 @@
     border-radius: 50%;
 }
 
-.add-area-button {
-    
+.input-lights-add-button-container {
+    display: flex;
+}
+
+.area-input-sectio{
```

Comment:
Hi Ola, you are missing the last letter at section. Can you complete the word "section" please. Thanks a lot.